include/fdreadoutlibs/wib/RAWWIBTriggerPrimitiveProcessor.hpp

```diff
@@ -13,7 +13,7 @@
 #include "readoutlibs/models/TaskRawDataProcessorModel.hpp"
 
 #include "fdreadoutlibs/FDReadoutTypes.hpp"
-#include "detdataformats/wib/RawWIBTp.hpp"
+#include "detdataformats/fwtp/RawTp.hpp"
```

Comment:
why this dependency on changes in detdataformats?  the memory leak could be fixed without this.